tests/autopilot-capacity/aut_test.go

```diff
@@ -33,71 +95,128 @@ var _ = BeforeSuite(func() {
 	InitInstance()
 })
 
-// This test performs basic test of starting an application, fills up the volume with data
-// which is more than the size of volume and waits that volume should be resized.
-var _ = Describe(fmt.Sprintf("{%sBasic}", testNameSuite), func() {
+// This testsuite is used for performing basic scenarios with Autopilot rules where it
+// schedules apps and wait until workload is completed on the volumes and then validates
+// PVC sizes of the volumes
+var _ = Describe(fmt.Sprintf("{%sWaitForWorkload}", testNameSuite), func() {
 	It("has to fill up the volume completely, resize the volume, validate and teardown apps", func() {
 		var contexts []*scheduler.Context
 		var err error
 		testName := strings.ToLower(fmt.Sprintf("%sBasic", testNameSuite))
```

Comment:
you renamed the test from `Basic` to `WaitForWorkload`. So this needs to get named too.